### DISCUSSION TITLE
Want to encrypt the blade file

### DISCUSSION THREAD

**Discussion by vishalmicrolevel** [laravel/framework#45061] [General]
I want to set up code for my client's desktop. but he is a little bit knowledgeable about Laravel.
So I don't want to give the original code to him. otherwise, he will sell my code also.
So I want the encrypt full Laravel code.
Using this package https://github.com/SiavashBamshadnia/Laravel-Source-Encrypter <= Encrypt PHP code.
but I m not able to encrypt the blade file code.
So using the blade file anyone knows this variable whose value is stored? so I don't want them.
So please give me a proper solution

**Comment by Sophist-UK** [comment]
@vishalmicrolevel You have already asked this in #44916 so this new discussion is effectively **spam**.

Please see #44916 for answers.
